src/services/infrastructure/entity-resolver/community.resolver.service.ts

```diff
@@ -149,12 +150,13 @@ export class CommunityResolverService {
         community: true,
       },
     });
+    // check for whiteboard in framing
     if (!space) {
       space = await this.entityManager.findOne(Space, {
         where: {
           collaboration: {
             callouts: {
-              contributions: {
+              framing: {
```

Comment:
That won't cover all cases, it's either under `framing`, or under `contributions`.
I think this breaks the latter case.

Reply:
The contributions check is right above this check. The issue was that the code was checking both times in contributions.